tests/test_waf_cli.py

```diff
@@ -122,6 +124,7 @@ def test_custom_waf_file_not_found(alias_session):
     assert result.exit_code == 0
 
 
+@pytest.mark.skip("The WAF logic is broken and needs to be remediated in another PR")
```

Comment:
Is that under the same ticket or can we reference the ticket here?

Reply:
I think this is the PR that resolves this logic issue -> https://github.com/uktrade/copilot-tools/pull/128.

Reply:
Gabe has solved this in another PR.  Either we get this PR merged and rebased or work the other way around.

Reply:
I think whoever gets their pull request in first gets it in first, we have quite a few open now, need to start getting them merged.

Reply:
@gabelton won 🤣